### DISCUSSION TITLE
Have a Sales Summary page

### DISCUSSION THREAD

**Discussion by luis-cruzt** [medusajs/medusa#5955] [Feature Requests]
The addition of a Sales Summary Page would be great, this page would allow to conveniently view and analyze the sales data.

Here's an example:
Sylius Dashboard
<img width="1435" alt="image" src="https://github.com/medusajs/medusa/assets/55716036/b46c1395-c25c-4c88-aab2-d182fb640567">

**Comment by olivermrbl** [comment]
Thanks for filing the request.

Until we have the capacity to look into a native dashboard, I recommend exploring our [Admin Extensions](https://docs.medusajs.com/admin/quickstart) and other tools to add custom functionality to projects. These allow you to build a sales dashboard that fits your exact needs.

E.g. use
- A [custom service](https://docs.medusajs.com/development/services/overview) to query data from other core services in Medusa to calculate your sales data
- A [custom endpoint](https://docs.medusajs.com/development/api-routes/overview) to expose the sales data
- A [UI Route](https://docs.medusajs.com/admin/routes) to query the endpoint and display the sales data

**Reply by luis-cruzt** [reply]
Thanks for the answer.

I'll try to do it. Do you have any advice on where (e.g., Product Service, Order Service) I need to query the data for my custom service in order to calculate the sales data?

**Comment by radoslaw-sz** [comment]
hi @luis-cruzt  - you may want to check: https://github.com/RSC-Labs/medusa-store-analytics

**Reply by luis-cruzt** [reply]
Wow! I'll take a look.

Thanks!
